### ISSUE TITLE
Live preview pane not the same as on projector

### ISSUE THREAD

**Issue by RootUserSam** [quelea-projection/Quelea#116]
When I put a verse or song in the live, the preview pane does not match what the congregation is seeing on the projector, which is screen 2. I have attached two screenshots to show what I mean.
What is seen on the projector:
![IMG_20190413_081927](https://user-images.githubusercontent.com/49592421/56082112-ab952a00-5dc9-11e9-83d0-9d5240a7633e.jpg)

What is seen on the main computer screen:
![IMG_20190413_081956](https://user-images.githubusercontent.com/49592421/56082116-b94aaf80-5dc9-11e9-972b-999943bcae0a.jpg)
As you can see, the preview pane and what is on the projector do not match. I need them to match no matter what. Please fix. Thank you.

**Comment by berry120** [comment]
Making them match "no matter what" is a challenge due to differing aspect
ratios - we'd have to change the aspect ratio of the preview screens based
on the current aspect ratio of the connected display screen, otherwise
there's always the chance they'll be out of alignment.

That's not something we do at the moment, but we'd certainly consider it
for a later release.

**Comment by PaulD987** [comment]
This one has caught my eye... From a quick look at the projection screen and preview/live screens on my machine the ratios look similar (if not the same), but what I have observed is if I resize the preview/live windows then the text reflows and wraps differently, although the ratio of the window should be the same still. I will do some more investigation as to what is going on here!

**Comment by DeeTigIT** [comment]
Was this solved in Quelea 2021 release? In that case this issue can be closed.  :-)
(I can reproduce it in Q 2020, but not in Q 2021)

**Comment by berry120** [comment]
@DeeTigIT I think so, but not entirely sure as my memory isn't great! We'll close it off but can reopen if it's still a problem.

**Comment by PaulD987** [comment]
Not sure it was completely fixed.... I delivered a fix under PR #332 which I think made things a lot better. I made a comment in the PR that it made things better, but wasn't a complete fix (but I can't remember now what I thought was still missing!). In any case, I think it is considerably better now, so maybe worth keeping closed unless someone can come up with an exact use case where it doesn't get it right!
